### ISSUE TITLE
Binance-0.2.0-alpha29: Error RateLimiter.DelayAsync count must not be less than 1

### ISSUE THREAD

**Issue by rakewell** [sonvister/Binance#54]
This is happening on the following line:

`Dim getOrders = Me.m_BinanceApi.GetOpenOrdersAsync(Me.m_BinanceUser)`

m_BinanceUser is instantiated with (don't think I needed to create an instance of ApiRateLimiter in this release but I tried anyway):

```
    Private Sub InitBinanceAccountUser()
        Me.m_BinanceUser = New BinanceApiUser(pubKey, privKey, New ApiRateLimiter)
    End Sub
```

{"RateLimiter.DelayAsync count must not be less than 1." & vbCrLf & "Parameter name: count"}
    Data: {System.Collections.ListDictionaryInternal}
    HResult: -2147024809
    HelpLink: Nothing
    InnerException: Nothing
    Message: "RateLimiter.DelayAsync count must not be less than 1." & vbCrLf & "Parameter name: count"
    ParamName: "count"
    Source: "Binance"
    StackTrace: "   at Binance.Api.RateLimiter.<DelayAsync>d__12.MoveNext() in D:\Dev\Internal\MessageStick.MessStockBot\StockDataManager.Dependencies\BinanceApi\Binance-0.2.0-alpha29\Api\RateLimit\RateLimiter.cs:line 65" & vbCrLf & "--- End of stack trace from previous location where exception was thrown ---" & vbCrLf & "   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)" & vbCrLf & "   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)" & vbCrLf & "   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable.ConfiguredTaskAwaiter.GetResult()" & vbCrLf & "   at Binance.Api.ApiRateLimiter.<DelayAsync>d__10.MoveNext() in D:\Dev\Internal\MessageStick.MessStockBot\StockDataManager.Dependencies\BinanceApi\Binance-0.2.0-alpha29\Api\RateLimit\ApiRateLimiter.cs:line 98" & vbCrLf & "--- End of stack trace from previous location where exception was thrown ---" & vbCrLf & "   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Tas
k task)" & vbCrLf & "   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)" & vbCrLf & "   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable.ConfiguredTaskAwaiter.GetResult()" & vbCrLf & "   at Binance.Api.BinanceHttpClientExtensions.<GetOpenOrdersAsync>d__17.MoveNext() in D:\Dev\Internal\MessageStick.MessStockBot\StockDataManager.Dependencies\BinanceApi\Binance-0.2.0-alpha29\Extensions\BinanceHttpClientExtensions.cs:line 642" & vbCrLf & "--- End of stack trace from previous location where exception was thrown ---" & vbCrLf & "   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)" & vbCrLf & "   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)" & vbCrLf & "   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()" & vbCrLf & "   at Binance.Api.BinanceApi.<GetOpenOrdersAsync>d__42.MoveNext() in D:\Dev\Internal\MessageStick.MessStockBot\StockDa
taManager.Dependencies\BinanceApi\Binance-0.2.0-alpha29\Api\BinanceApi.cs:line 600"
    TargetSite: {Void MoveNext()}

**Comment by sonvister** [comment]
@rakewell, thanks for mentioning. That method as well as `Get24HourStatisticsAsync` use the number of 'trading' symbols to compute the rate limit weight. I suspect that number was 0 at the time and caused the exception. Trading is active again and these methods should not fail now, but I will fix that issue and have it in the next release.

**Comment by sonvister** [comment]
@rakewell, the methods rely on the Symbol cache that is currently initialized with 0 symbols trading (*status at the time of the build*). So, the methods will still fail unless the Symbol cache is updated.

To update the cache: `Symbol.UpdateCacheAsync(<IBinanceApi>)`

**Comment by sonvister** [comment]
The fix has been released in **0.2.0-alpha30**.

**Comment by rakewell** [comment]
awesome, thanks @sonvister no error in the alpha30 release.
